### PR TITLE
add new section to Service Registry getting started on assigning role to service account

### PR DESCRIPTION
**Pull request by smccarthy-ie**
@wtrocki @EricWittmann @carlesarnal This PR makes it clear that you must assign a role to a service account for client applications. Please review for technical accuracy and let me know if you approve or have any more updates. Thanks.

### CODE REVIEW

File: getting-started-service-registry/README.adoc [hunk 1]
```diff
@@ -147,10 +144,10 @@ NOTE:  Specifying a group is optional when using the web console, and a `default
...
+* *Info*: Displays the artifact name, group, description, lifecycle status, when created, and last modified. 
 * *Content*: Displays a read-only view of the full artifact content.
 * *Documentation*: (OpenAPI only): Displays automatically-generated REST API documentation.
-* *Content Rules*: Displays artifact content rules that you can enable and configure. You can configure a *Validity Rule* or *Compatibility Rule* by selecting the appropriate rule configuration from the drop-down. For details on supported rules, see the https://access.redhat.com/documentation/en-us/red_hat_openshift_service_registry/1[{registry-product-long} user documentation].
+* *Content Rules*: Displays artifact content rules that you can enable and configure. You can configure a *Validity Rule* or *Compatibility Rule* by selecting the appropriate rule configuration from the drop-down. For details on supported rules, see the https://access.redhat.com/documentation/en-us/red_hat_openshift_service_registry/1[{registry-product-long} user documentation^].
```
Comment (wtrocki) [inline getting-started-service-registry/README.adoc:150]: Is this desired?

**Review by wtrocki — APPROVED** [review]

File: getting-started-service-registry/README.adoc (hunk 1 above)
Comment (smccarthy-ie) [reply getting-started-service-registry/README.adoc:150]: @wtrocki If you mean the ^ caret character, yes to open link in new tab

Comment (wtrocki) [reply getting-started-service-registry/README.adoc:150]: Nice!

**Review by EricWittmann — APPROVED** [review]
LGTM

**Comment by wtrocki** [comment]
Done another pass and verified it end to end.
I think we can merge that.

**Comment by wtrocki** [comment]
We should still have QA after all is published to the docs

**Comment by smccarthy-ie** [comment]
Thanks @wtrocki  I've pinged QE on the appropriate Jira and copied you.

**Pull request merged**